package.json

```diff
@@ -33,7 +33,6 @@
     "@canonical/react-components": "1.7.1",
     "@monaco-editor/react": "4.6.0",
     "@tanstack/react-query": "5.51.23",
-    "@use-it/event-listener": "0.1.7",
```

Comment:
why are we removing this dependency?

Reply:
The types for that library are incompatible, and it seemed simple enough to  implement  ourselves